src/libertem/api.py

```diff
@@ -91,9 +91,10 @@ def create_mask_job(self, factories, dataset):
         return ApplyMasksJob(
             dataset=dataset,
             mask_factories=factories,
+            use_sparse=use_sparse
         )
 
-    def create_mask_analysis(self, factories, dataset):
+    def create_mask_analysis(self, factories, dataset, use_sparse=None):
```

Comment:
As discussed, add `use_sparse` to docstring.

Reply:
Done! :-)